spark/src/main/scala/geotrellis/spark/io/accumulo/AccumuloLayerUpdater.scala

```diff
@@ -32,7 +32,7 @@ class AccumuloLayerUpdater[K: Boundable: JsonFormat: ClassTag, V: ClassTag, M: J
       case e: UnsupportedOperationException => throw new LayerUpdateError(id, ": empty rdd update").initCause(e)
```

Comment:
Probably we can avoid catching this exception: it is workaround for the `getKeyBounds` function applied to an empty rdd.